CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Confluent's Golang client for Apache Kafka
 
+# v2.0.0
+
+This is a maintenance release:
+
+* Channel based producer (`.ProduceChannel()`) and channel based consumer (`.Events()`) are deprecated.
```

Comment:
Since both Producer and Consumer has an Events() (iirc), I think it might be good to actually say something like
```suggestion
* Channel based producer (Producer `ProduceChannel()`) and channel based consumer (Consumer `Events()`) are deprecated.
```